xExtension-CustomCSS/extension.php

```diff
@@ -19,9 +19,16 @@ public function handleConfigureAction() {
 
 		$current_user = Minz_Session::param('currentUser');
 		$filename =  'style.' . $current_user . '.css';
-		$filepath = join_path($this->getPath(), 'static', $filename);
+		$staticPath = join_path($this->getPath(), 'static');
```

Comment:
Oh there's a new path thing. On phone atm.

Reply:
Found it https://github.com/FreshRSS/FreshRSS/issues/1456#issuecomment-348797338

Reply:
Yes I saw that, but a) its for 1.9 and b) its not usable right now: https://github.com/FreshRSS/FreshRSS/issues/1709

Reply:
Alright, cool.